File: parallelepiped.c
#include <stdio.h>

int main()
{
    float a,b,c,S,V;
    printf("Enter value of a: ");
    scanf("%f", &a);
    printf("Enter value of b: ");
    scanf("%f", &b);
    printf("Enter value of c: ");
    scanf("%f", &c);
    if(a <= 0 || b <= 0 || c <= 0){
    printf("Invalid parallelepiped!\n");
    return 1;
    }
    S = 2 * (a * c + b * c);
    printf("The parallelepiped's area is %.2f!\n", S);
    V = a * b * c;
    printf("The parallelepiped's volume is %.2f!\n", V);
    return 0;
}

File: parallelepiped.py
a = float(input('Enter value of a: '))
b = float(input('Enter value of b: '))
c = float(input('Enter value of c: '))
if a <= 0 or b <= 0 or c <= 0:
    print('Invalid parallelepiped!')
else:
    S = 2 * (a * c + b * c)
    formatedArea = "{:.2f}".format(S)
    V = a * b * c;
    formatedVolume = "{:.2f}".format(V)
    print("Trapezoid's area is " + str(formatedArea) + "!")
    print("Trapezoid's volume is " + str(formatedVolume) + "!")
